test/jasmine/tests/mapbox_test.js

```diff
@@ -1226,11 +1226,18 @@ describe('@noCI, mapbox plots', function() {
             expect(layout.zoom).toBeCloseTo(zoom);
         }
 
-        function _assert(center, zoom) {
+        function _assert(center, zoom, lonMin, latMin, lonMax, latMax) {
```

Comment:
`lonMin` could sometimes be greater than `lonMax`.
What about using `lon1`, `lon2`, `lat1` and `lat2` here?

Reply:
:+1:

Reply:
Done in 4b2623fda84e818e600c9539ea45b3ce80b558ba